src/fileActions.jsx

```diff
@@ -474,10 +474,19 @@ export const fileActions = (path, selected, setSelected, clipboard, setClipboard
     const spawnPaste = (sourcePath, targetPath) => {
         cockpit.spawn([
             "cp",
-            "-R",
+            "--archive",
+            "--recursive",
+            "--interactive",
             ...sourcePath,
             targetPath
-        ]).catch(err => addAlert(err.message, "danger", new Date().getTime()));
+        ], { superuser: "try", environ: ["LC_ALL=C"] }).input()
```

Comment:
Somehow GH ate my most important comment at all: This is a major security hole. If you copy a dir from e.g. /home/joe to /home/jane, or /tmp/download/ into /var/lib/postgresql, you give the source user a foot into the door of the target user. The CLI default is quite deliberately to copy files as root, to avoid this.

Reply:
What I mean to say is:

> The CLI default is quite deliberately to copy files as root, to avoid this.

"citation needed".

I know you're very much allergic to this "different owner than parent directory" thing but other very common coreutils commands do exactly what you're suggesting is so evil, by default.

Reply:
That creates the same problem of course, but indeed unfortunately there's no warning. Of course on the CLI you, shouldn't do that, but the CLI gives you all sorts of ways to shoot yourself into the foot.

Reply:
Also, pasting something feels more like a clipboard, and it's not at all obvious during that operation what the source/target owner will be. `mv /home/joe /home/jane` is at least very explicit.